Restructuring/projects handout (#414)
neuefische/web-exercises: sessions/recap-project-3/rick-and-morty-app/README.md

# Recap Project 3: Rick and Morty App

In this project, you will create a single-page app for browsing all characters from the popular TV show **Rick and Morty**.

This is a group project, so please create **only one project per group** following the [guide in the template section](./README.md#template).

## Overview

> This single-page app will allow users to browse all characters from *Rick and Morty*. The app will include a search input to filter characters, and all relevant results should display after a search. If the search results span multiple pages, users should be able to navigate through the pages using "Next" and "Previous" buttons.

Reviewer: ```suggestion
This single-page app will allow users to browse all characters from *Rick and Morty*. The app will include a search input to filter characters and all relevant results should be displayed after a search. A basic pagination system with "Next" and "Previous" navigation buttons lets the user browse large numbers of results with ease.
```

> 💡 You might run into some bugs at this point. Think about how the page and max page index might
> have to change when you start searching for only subsets of all characters.

> ## Bonus

Reviewer: ```suggestion
### Bonus: Refactoring your Code
```

Author: It was decided to keep the Bonus as h2.
Task are mandatory feature, meanwhile bonus and bonus, extra

- Append the created components at the right places in your HTML. All container elements are already
  available in the `index.js`.

> ### Style Your Project

Reviewer: ```suggestion
### Bonus: Style Your Project
```
